sdk/logical/storage_transactions.go

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) OpenBao, Inc.
```

Comment:
Are you sure that this is correct? For the website I was asked to use the following:
> Copyright © 2024 OpenBao a Series of LF Projects, LLC

Reply:
Interesting, I didn't know that we had gotten clarity about that. Happy to update that.

Reply:
Interestingly OpenTofu uses
> Copyright (c) The OpenTofu Authors
According to https://github.com/opentofu/opentofu/issues/454#issuecomment-1904213569 this seems to have been discussed with their lawyers. So maybe we could also adopt a similar wording

Reply:
I'm going to with:

`Copyright (c) 2024 OpenBao a Series of LF Projects, LLC`

since I don't want to copy/paste the special © every time.